ypkg2/main.py

```diff
@@ -13,6 +13,7 @@
 
 from . import console_ui
 from . import remove_prefix
+from .compressdoc import compress_info_pages, compress_manpages
```

Comment:
Nit: inconsistent function names info_pages vs manpages

Reply:
Which way should these go? I figured they were called manpages and info pages, so I was trying to go with what they're usually called.

Reply:
i mean info pages has an underscore between it and man pages doesn't in the function name.

Reply:
Yes, I just thought that manpages was one word usually.

Reply:
eh, i don't really mind either way

Reply:
I don't either xD I'll just add an underscore.